go-controller/pkg/network-controller-manager/network_controller_manager.go

```diff
@@ -66,7 +67,7 @@ func (cm *NetworkControllerManager) NewNetworkController(nInfo util.NetInfo) (na
 	case ovntypes.Layer3Topology:
 		return ovn.NewSecondaryLayer3NetworkController(cnci, nInfo)
 	case ovntypes.Layer2Topology:
-		return ovn.NewSecondaryLayer2NetworkController(cnci, nInfo), nil
+		return ovn.NewSecondaryLayer2NetworkController(cnci, nInfo)
```

Comment:
on commit "L2 services support" it would be good to have a commit message describing in general what you did to make it work